Tests/Admin/AdminHelperTest.php

```diff
@@ -25,16 +25,16 @@ class AdminHelperTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $container = $this->getMock('Symfony\Component\DependencyInjection\ContainerInterface');
+        $container = $this->createMock('Symfony\Component\DependencyInjection\ContainerInterface');
```

Comment:
Good point, thank you!
At first I want to suggest use trait, but then I remember about supporting php 5.3. Will fix it